exonum-java-binding-common/pom.xml

```diff
@@ -88,6 +88,10 @@
       <artifactId>equalsverifier</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
```

Comment:
Why do we need it ? 
if I understand correctly you add it to use assertThat() ? But we already use assertThat() from  org.hamcrest.MatcherAssert.assertThat  in common tests